packages/components/table/src/store/watcher.ts

```diff
@@ -118,15 +118,19 @@ function useWatcher<T>() {
       (column) =>
         column.type !== 'selection' && [true, 'left'].includes(column.fixed)
     )
-
+    // 找有fixed的selectColumn
```

Comment:
The comments here can be removed, the variable names and code logic are clear enough.